addons/base_vat/models/res_partner.py

```diff
@@ -23,6 +23,9 @@
 
 _eu_country_vat_inverse = {v: k for k, v in _eu_country_vat.items()}
 
+_eu_country_incorrect_vat = [k for k,v in _eu_country_vat.items()]
+_eu_country_correct_vat = [v for k,v in _eu_country_vat.items()]
```

Comment:
i'm wondering if those are not really needed since you can use _eu_country_vat_inverse.keys() and .values(). Could you try with that ?